### PR TITLE
Fix font-awesome version to 4.7.0

### PR DESCRIPTION
**Pull request by taiwo-ladipo**
Font-awesome 5.0 has been released and sadly does not have the same directory structure as the older versions. So, it is safe to lock it on 4.7.0 for now

### CODE REVIEW

**Review by goke-epapa — APPROVED** [review]

File: composer.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
     "require": {
         "php" : "~5.5|~7.0",
         "yiisoft/yii2": ">=2.0.8",
-        "bower-asset/font-awesome": ">=4.7.0",
+        "bower-asset/font-awesome": "4.7.0",
```
Comment (yemexx1) [inline composer.json]: Let's take fixes @taiwo-ladipo

**Review by otaruMendez — APPROVED** [review]

File: composer.json (hunk 1 above)
Comment (goke-epapa) [reply composer.json]: Yeah and Minor updates too

Comment (taiwo-ladipo) [reply composer.json]: Done @yemexx1 👍

**Review by yemexx1 — APPROVED** [review]

File: composer.json [hunk 2]
```diff
@@ -19,7 +19,7 @@
     "require": {
         "php" : "~5.5|~7.0",
         "yiisoft/yii2": ">=2.0.8",
-        "bower-asset/font-awesome": ">=4.7.0",
+        "bower-asset/font-awesome": "~4.7.0",
```
Comment (otaruMendez) [inline composer.json:22]: Can we use ^4.7.0 instead or ~4 to include minor version @taiwo-ladipo

**Pull request merged**